### ISSUE TITLE
CommandBar: Narrator does not announce item click event

### ISSUE THREAD

**Issue by v-olvoli** [OfficeDev/office-ui-fabric-react#11395]
### Environment Information

- **Package version(s)**: ("office-ui-fabric-react": "^7.54.1")
- **Browser and OS versions**: (Microsoft Edge 44.18362.449.0 and Windows 10)

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/v-olvoli/pen/RwNWWKZ

#### Actual behavior:
Narrator only announces text specified in the ariaLabel property when CommandBar item gets focus.

When you navigate to the CommandBar item for which ariaLabel property is specified (using keyboard: Tab and Arrows) and then press Enter or Space key - Narrator does not announce that action has been triggered.

#### Expected behavior:
When you navigate to the CommandBar item for which ariaLabel property is specified (using keyboard: Tab and Arrows) and then press Enter or Space key - Narrator should announce that action has been triggered.

For example:
When user presses Enter or Space key on the CommandBar item with ariaLabel: 'Refresh' then this should be announced as 'Refresh has been triggered' or 'Refresh button has been clicked' or something like this.

**Comment by xugao** [comment]
@v-olvoli - in case you added `ariaLabel` to the command item, it's behaving as expected. That is, when item is focused, aria label is read out. 
For the case when you click (space/enter key) on the item, what you need to do is to change the focus onto the next thing triggered by the click action. For instance, if window refreshed, focus will go back onto the window and narrator will start read out the site content.

**Comment by v-olvoli** [comment]
@xugao , thanks for the quick response, I've got it!
Let me please clarify what is the problem I've faced with and why I can't use your suggestion.
On the project I'm working on we have a lot command buttons such as 'Refresh', 'Next page', 'Previous page', 'Settings', 'Filters', 'Legend', 'Select date' and so on... and according to the requirements we need to announce exact action (click) that is performed by customers that are using screen readers (i.e. "refresh has been performed" or at least "refresh").
I know office-ui-fabric-react can announce such actions via this functionality - https://developer.microsoft.com/en-us/fabric#/controls/web/announced/quickactions
But this should be implemented for each button separately that will require us to write much more code than in case if we would specify some property (something like ariaLabel) for the button to be announced by screen reader.
Do you have such property or maybe you plan to implement this in the future?
Thanks!

**Comment by xugao** [comment]
@v-olvoli - Ah ic. this is not something we support from CommandBar directly. you would need to use quickactions in this case. 
we don't have plan to add this to any individual component and it's more suitable as a utility that we provide for users to apply on any other components

**Comment by v-olvoli** [comment]
@xugao , thank You!
